Description: update POD to point to the package that actually contains ysh
Origin: vendor
Forwarded: not-needed
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-04-09

--- a/lib/YAML.pm
+++ b/lib/YAML.pm
@@ -748,7 +748,7 @@
 
 =head1 ysh - The YAML Shell
 
-The YAML distribution ships with a script called 'ysh', the YAML shell.
+libyaml-shell-perl contains a script called 'ysh', the YAML shell.
 ysh provides a simple, interactive way to play with YAML. If you type in
 Perl code, it displays the result in YAML. If you type in YAML it turns
 it into Perl code.
